app/controllers/pieces_controller.rb

```diff
@@ -213,7 +213,16 @@ def check_game_status
 
     # Methods to call
     # Checkmate, lose or win
-    # Check
+    # Check with in_check? method
+    if game.in_check?(color)
+      add_player_messages('Ugh... Your king is in check!', 'Nice! You\'ve put opponent\'s king in check!')
+      check_game_status
+      @game.update_player_turn
+      200
```

Comment:
Don't need the 200 or 422 status returns here. def check_game_status is only to override the player messages. So really all you'll need is `if @game.in_check? add_player_messages(etc, etc) elsif #add next game check here end`